src/Components/Search/NewSearch/Mobile/Overlay.tsx

```diff
@@ -13,13 +12,17 @@ import createLogger from "Utils/logger"
 import { NewSearchInputPillsFragmentContainer } from "Components/Search/NewSearch/NewSearchInputPills"
 import { reportPerformanceMeasurement } from "Components/Search/NewSearch/utils/reportPerformanceMeasurement"
 import { shouldStartSearching } from "Components/Search/NewSearch/utils/shouldStartSearching"
-import { useDebounce } from "Utils/Hooks/useDebounce"
-import { useTracking } from "react-tracking"
-import * as DeprecatedSchema from "@artsy/cohesion/dist/DeprecatedSchema"
+import { useDebounce, useDebouncedValue } from "Utils/Hooks/useDebounce"
 import { MobileSearchBar_viewer$data } from "__generated__/MobileSearchBar_viewer.graphql"
+import { OverlayBase } from "Components/Search/NewSearch/Mobile/OverlayBase"
+import { SearchResultsListQueryRenderer } from "Components/Search/NewSearch/Mobile/SearchResultsList"
 
 const logger = createLogger("Components/Search/NewSearch/Mobile")
 
+const scrollToTop = () => {
```

Comment:
Can we use this? https://github.com/artsy/force/blob/main/src/Utils/Hooks/useJump.tsx

Reply:
Also @dzucconi - do you think `useJump` might be less discoverable than `useScrollTo`? There's also a few other `useScrollToThing` hooks in the utils/hooks folder

Reply:
useJump is useful if we need to accommodate sticky items and/or nav. I don't see why we can't simply `window.scrollTo(0, 0)` here?

Reply:
Cool, np then 👍